apps/beeswax/src/beeswax/api.py

```diff
@@ -15,9 +15,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import re
 import json
 import logging
+import re
```

Comment:
Note that this and several other non functional changes were auto fixed by the linter and added in a separate commit.